components/partials/swap/index.vue

```diff
@@ -0,0 +1,1574 @@
+<template>
```

Comment:
this component is too big with **1500 lines** of codes, lets try to refactor this into small components